src/components/drag-and-drop/draggable-context/draggable-context.js

```diff
@@ -109,6 +131,59 @@ class DraggableContext extends React.Component {
    */
   handleHover = ItemTargetHelper.onHoverUpDown
 
+  /**
+   * A callback for when auto scroll is triggered
+   *
+   * @method autoScroll
+   * @param {Object} ev event
+   * @return {void}
+   */
+  checkAutoScrollTrigger = (event) => {
+    if (!this.state.dragging) {
+      return;
+    }
+
+    // constant for the position of the mouse pointer
+    // on the y axis
+    const { clientY } = event;
+
+    // constant for the threshold where the auto scroll begins
+    const threshold = Math.max(140, window.innerHeight / 4);
+
+    let speed = 0;
+    if (clientY < threshold) {
+      // -1 to 0 as we move from 0 to threshold
+      speed = -1 + clientY / threshold;
+    } else if (clientY > window.innerHeight - threshold) {
+      // 0 to 1 as we move from (innerHeight - threshold) to innerHeight
+      speed = 1 - (window.innerHeight - clientY) / threshold;
+    } else {
+      speed = 0;
+    }
+
+    if (this.speed === 0 && speed !== 0) {
+      this.speed = speed;
+      this.startScrolling();
+    }
+    this.speed = speed;
+  }
+
+  startScrolling = () => {
+    if (!this.frame) {
+      this.frame = window.requestAnimationFrame(this.tick);
+    }
+  }
+
+  tick = () => {
+    if (!this.speed) {
+      this.frame = null;
+      return;
+    }
+
+    window.scrollTo(0, window.scrollY + (this.speed * 10));
+    this.frame = window.requestAnimationFrame(this.tick);
```

Comment:
should this be `this.startScrolling()`?

Reply:
That won't work. I believe that `startScrolling` is only used when the scrolling needs to be initiated when there is no `frame` object. When swapping that line with a call to the startScrolling method, it breaks the whole auto scroll,